src/web/modules/transactions/discrepancies/validateLedger.http.ts

```diff
@@ -13,39 +15,79 @@ export async function validateLedgerTransaction(
 ): Promise<void> {
   try {
     const ledgerEntry = await Ledger.transactions.retrieve(req.params.id)
-    const connectorEntry = await Connector.charges.retrieveAPI(req.params.id, ledgerEntry.gateway_account_id)
+    const connectorEntry = await Connector.charges.parityCheck(req.params.id, ledgerEntry.gateway_account_id)
+    let warningMessage
+    if (typeof connectorEntry === 'string') {
+        warningMessage = connectorEntry
```

Comment:
I don't think this needs to be a warning, it's expected behaviour for a charge to not exist in Connector once it has been expunged.

This should also be reflected in the view as 'info' rather than a 'problem'

For actual differences we should show a warning

Reply:
Good point, changed

Reply:
<img width="495" alt="Screenshot 2024-09-19 at 12 55 26" src="https://github.com/user-attachments/assets/a96df6df-823e-4d49-8706-0d69e36ccb0a">